docker-compose.yaml

```diff
@@ -10,6 +10,9 @@ services:
       - 3000
     env_file:
       - ./api/.env
+    environment:
+      DATABASE_URL: file:./db.sqlite
+      PRISMA_CLIENT_OUTPUT: "/app/.output/server/node_modules/.prisma/client"
```

Comment:
This is path is valid for nuxt, but this service (`api`) doesn't use nuxt or build anything into `.output` folder, so the path is ultimately incorrect.  Or does prisma recursively creates all those folders that doesn't exist and doesn't care about them? Maybe specifying it didn't really helped here, but instead created another problem

Reply:
Yeah that seems to be the case so i removed it for now. Unfortunatly after a lot of try and error im still not sure what values needs to be added.